test/helpers/cons.go

```diff
@@ -243,6 +244,33 @@ const (
 	failedToUpdateLock         = "Failed to update lock:"
 	failedToReleaseLock        = "Failed to release lock:"
 	errorCreatingInitialLeader = "error initially creating leader election record:"
+	cantEnableJIT              = "bpf_jit_enable: no such file or directory"                               // Because we run tests in Kind.
+	delMissingService          = "Deleting no longer present service"                                      // cf. https://github.com/cilium/cilium/issues/29679
+	failedIpcacheRestore       = "Failed to restore existing identities from the previous ipcache"         // cf. https://github.com/cilium/cilium/issues/29328
+	podCIDRUnavailable         = " PodCIDR not available"                                                  // cf. https://github.com/cilium/cilium/issues/29680
```

Comment:
nit: seems like the space prefix can be removed.

Reply:
Maybe a bit weird, but it's actually on purpose. The full message is:
```
level=warning msg="Waiting for k8s node information" error="required IPv4 PodCIDR not available" subsys=daemon
```
I was therefore trying to make it as specific to this specific error as possible, but also wanted to match on the equivalent for IPv6. "PodCIDR not available" is fairly generic so I figured `error="PodCIDR not available"` might exist somewhere. The space ensures that's not the full message.

So yeah, bit weird, but hopefully fine since those strings are just ways to match on allowlisted warnings.